src/test/java/io/reactivex/plugins/RxJavaPluginsTest.java

```diff
@@ -43,11 +32,22 @@
 import io.reactivex.internal.operators.observable.ObservableRange;
 import io.reactivex.internal.operators.parallel.ParallelFromPublisher;
 import io.reactivex.internal.operators.single.SingleJust;
-import io.reactivex.internal.schedulers.ImmediateThinScheduler;
+import io.reactivex.internal.schedulers.*;
```

Comment:
Now that this doesn't have an extra method, please restore the pre-PR version.

Reply:
done